tools/sigma/backends/lacework.py

```diff
@@ -62,6 +61,73 @@
           - EVENT_TIME
           - EVENT
         alertProfile: LW_CloudTrail_Alerts
+        evalFrequency: Hourly
+    product.categories:
+      linux.file_create:
+        evaluatorId:
+        source: LW_HE_FILES
+        conditions:
+          # evaluated hourly and file create time within the last hour
+          - and diff_minutes(FILE_CREATED_TIME, current_timestamp_sec()::timestamp) <= 60
+        fieldMap:
+          - sigmaField: TargetFilename
+            laceworkField: PATH
+            matchType: exact
+        returns:
+          - RECORD_CREATED_TIME
+          - MID
+          - PATH
+          - FILE_TYPE
+          - SIZE
+          - FILEDATA_HASH
+          - OWNER_UID
+          - OWNER_USERNAME
+          - FILE_CREATED_TIME
+        alertProfile: LW_HE_Files.HE_File_NewViolation
+        evalFrequency: Hourly
+      linux.process_creation:
+        evaluatorId:
+        source: LW_HE_PROCESSES
+        conditions:
+          # evaluated hourly and file create time within the last hour
+          - and diff_minutes(PROCESS_START_TIME, current_timestamp_sec()::timestamp) <= 60
+        fieldMap:
+          - sigmaField: ParentImage
+            laceworkField:
+            matchType: exact
+            action: raise
+          - sigmaField: Image
+            laceworkField: EXE_PATH
+            matchType: exact
+          - sigmaField: ParentCommandLine
+            laceworkField:
+            matchType: exact
+            action: raise
+          - sigmaField: CommandLine
+            laceworkField: CMDLINE
+            matchType: exact
+          - sigmaField: CurrentDirectory
+            laceworkField: CWD
+            matchType: exact
+          - sigmaField: LogonId
+            laceworkField:
+            matchType: exact
+            action: ignore
+          - sigmaField: User
+            laceworkField: USERNAME
+            matchType: exact
+        returns:
+          - RECORD_CREATED_TIME
+          - MID
+          - PID
+          - EXE_PATH
+          - CMDLINE
+          - CWD
+          - ROOT
+          - USERNAME
+          - PROCESS_START_TIME
+        alertProfile: LW_HE_Files.HE_Process_NewViolation
```

Comment:
and this perhaps `LW_HE_PROCESSES_DEFAULT_PROFILE.HE_Process_NewViolation`

Reply:
Fixed.